substratevm/src/com.oracle.svm.jfr/src/com/oracle/svm/jfr/JfrManager.java

```diff
@@ -206,7 +193,7 @@ private static Long parseDuration(Map<JfrStartArgument, String> args, JfrStartAr
                     return Duration.ofMinutes(time).toNanos();
                 } else if ("h".equals(unit)) {
                     return Duration.ofHours(time).toNanos();
-                } else if ("f".equals(unit)) {
+                } else if ("d".equals(unit)) {
                     return Duration.ofDays(time).toNanos();
                 }
                 throw new IllegalArgumentException("Unit is invalid.");
```

Comment:
What does HotSpot do in this case? Does it ignore the invalid unit (like in `parseMaxSize` below) or throw an error?

Reply:
@christianhaeubl Hello,
HotSpotthrows an error in this case:
```
$ java -XX:+FlightRecorder -XX:StartFlightRecording=delay=5f  Main
java.lang.IllegalArgumentException: Integer parsing error nanotime value: illegal unit
Error occurred during initialization of VM
Failure when starting JFR on_create_vm_2
```

So the behavior is the same but for the error message wording.

Reply:
Thanks for checking!